package.json

```diff
@@ -15,7 +15,6 @@
     "draft-js": "^0.11.0",
     "draftjs-to-html": "^0.8.4",
     "fast-deep-equal": "^2.0.1",
-    "html-to-draftjs": "^1.4.0",
```

Comment:
This dependency will be required when setting an initial value on the rich text editor.

Reply:
That's good to know, I'll try to add an example to the storybook.